### ISSUE TITLE
Document additional dependencies for building the GUI

### ISSUE THREAD

**Issue by aspaas** [cevap/ion#33]
The ION GUI has several dependencies beyond the dependencies that are set based on Bitcoin compatibility. 
For example: Qt5PrintSupport is a dependency for ION, not for Bitcoin.
When updating the automake file, these dependencies need to be accounted for. At the same time, they need to be documented to:
- Update the installation process 
- Be aware of the differences when upgrading the code

**Comment by cevap** [comment]
Agree, let's discuss it and let's see how we manage this project, as it has with many different segments to do.

If you did not look up, take a look at [General Development Workflow Strategy Definition (Build-Release process ,now and in future, dependancies - static builds) ](https://github.com/cevap/ion/projects/7) (_simply click on "Show Menu" to see project description_) . You could also read the same in [projects overview page](https://github.com/cevap/ion/projects)
